src/main/java/edu/umich/ctools/sectionsUtilityTool/Utils.java

```diff
@@ -22,4 +31,129 @@ public static Properties loadProperties(String path){
 		M_log.error("File path for (" + path + ") is not provided");
 		return null;		
 	}
-}
+
+	public static void openFile(HttpServletRequest request,
+			HttpServletResponse response, PrintWriter out) {
+		M_log.debug("Stub method called");
+		String pathInfo = request.getPathInfo();
+		M_log.debug("Path Info: " + pathInfo);
+		
+		String url;
+		String queryString = request.getQueryString();
+		
+		if(queryString!=null) {
```

Comment:
Could be:
 url=pathInfo
if queryString != null then {
url+="?"+queryString
}